### PR TITLE
Taiwan Trad. Chinese Translation

### PR DESCRIPTION
**Pull request by kogiokka**
Taiwanese Mandarin localization (Traditional Chinese)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
+        classpath 'com.android.tools.build:gradle:3.1.4'
```
Comment (BO41) [inline build.gradle]: I don't think the gradle version needs to be updated just for a new translation.
Your IDE probably did this automatically

Comment (kogiokka) [reply build.gradle]: @BO41 Sorry if this causes trouble. Can the admins edit the change and merge the rest part, or should I do another Pull request?

Comment (BO41) [reply build.gradle]: I am not sure if this is a problem. I just think it shouldn't be updated with a translation PR.
@scoute-dich needs to decide. I am a fan of having dependencies and things like this up-to-date but sometimes this causes trouble.

If it is not wanted, just go into the file and change the number back an commit this change. No need for a new PR ;)

Comment (kogiokka) [reply build.gradle]: @BO41 Thanks for the reply and suggestion!

**Pull request merged**